pkg/runner/local_exec.go

```diff
@@ -44,7 +45,11 @@ type LocalExecutableRunner struct {
 }
 
 // LocalExecutableRunnerCfg is the configuration struct for this runner.
-type LocalExecutableRunnerCfg struct{}
+type LocalExecutableRunnerCfg struct {
+	// BinPath is the path of the binary relative to the artifact path. Use this
+	// if the builder doesn't put the binary as the artifact path
+	BinPath string `toml:"bin_path"`
```

Comment:
Just to confirm I understand this. We need this for nix. It outputs an artifact. We pass the artifact path to local exec runner. We want to execute a specific binary within that artifact. Is it correct?